### PR TITLE
Update interpolated-strings.md

### PR DESCRIPTION
**Pull request by iubns**
Correct grammatical errors

### CODE REVIEW

File: docs/csharp/language-reference/keywords/interpolated-strings.md [hunk 1]
```diff
@@ -19,12 +19,12 @@ Used to construct strings.  An interpolated string looks like a template string
...
   
 ```csharp  
-Console.WriteLine($"Name = {name}, hours = {hours:hh}"); 
+Console.WriteLine($"Name = {name}, hours = {hours}:hh"); 
 ```  
 contains two interpolated expressions, '{name}' and '{hours:hh}'. The equivalent composite format string is:
 
 ```csharp
-Console.WriteLine("Name = {0}, hours = {1:hh}", name, hours);  
+Console.WriteLine("Name = {0}, hours = {1}:hh", name, hours);
```
Comment (svick) [inline docs/csharp/language-reference/keywords/interpolated-strings.md]: I think the original code is correct: the `:hh` is a format string, not something that should be printed.

But the original code is also confusing: if it's supposed to print hours, then `hours` should be a `DateTime`. But `hours` is not a good name for a `DateTime` variable.

I would change the snippets to:

```c#
Console.WriteLine($"Name = {name}, hours = {date:hh}"); 
Console.WriteLine("Name = {0}, hours = {1:hh}", name, date);  
```

Comment (iubns) [reply docs/csharp/language-reference/keywords/interpolated-strings.md]: you are right!

Comment (BillWagner) [reply docs/csharp/language-reference/keywords/interpolated-strings.md]: @iubns Do you want to update this PR per @svick's comments?  If so, just add new commits and sync it again.

Thanks for the contribution. We really appreciate it.

**Review by svick — APPROVED** [review]

**Review by BillWagner — APPROVED** [review]
Thanks for spotting this and fixing it @iubns We appreciate it.

I'll approve this PR, and :shipit: now.  It will show up on the live site in a day or two, on our next regular publish cycle.

**Pull request merged**